src/web/demo.rkt

```diff
@@ -439,6 +440,26 @@
               'children '())])))
       (hasheq 'tree tree 'job id))))
 
+
+(define explanations-endpoint
+  (post-with-json-response
+    (lambda (post-data)
+      (define formula (read-syntax 'web (open-input-string (hash-ref post-data 'formula))))
+      (define sample (hash-ref post-data 'sample))
+      (define seed (hash-ref post-data 'seed #f))
+      (eprintf "Explanations job started on ~a..." formula)
+
+      (define test (parse-test formula))
+      (define expr (prog->fpcore (test-input test) (test-output-repr test)))
+      (define pcontext (json->pcontext sample (test-context test)))
+      (define result (run-herbie 'explanations test #:seed seed #:pcontext pcontext
+                                 #:profile? #f #:timeline-disabled? #t))
```

Comment:
I think you need to use the `job` API here—`start-job`, `wait-for-job`, `job-result-backend`.

Reply:
See one of the other endpoints for an example.